playwright.ts

```diff
@@ -31,7 +31,9 @@ class MyReporter implements Reporter {
     protected _initPromise: Promise<void>;
 
     constructor(opts: Partial<ReporterConfig>) {
-        this._config = parseConfig(_.omit(opts, ['configDir']));
+        const reporterOpts = _.omitBy(opts, (_value, key) => key === 'configDir' || key.startsWith('_'));
```

Comment:
we don't have unit tests for this, but there are integration tests. That's why I updated the version of pwt to the latest.